whatshap/cli/stats.py

```diff
@@ -341,13 +345,17 @@ def parse_variant_tables(vcf_reader, chromosomes=None):
         yield from vcf_reader
 
 
-def get_chr_lengths(chr_lengths_file, vcf_reader):
+def get_chr_lengths(vcf_reader: VcfReader, chr_lengths_file: Optional[str] = None) -> Dict[str, int]:
+    """
+    Try to map chromosome length to its name. Use chr_lengths_file if it exists.
+    Otherwice look in VCF header for chromosomes.
```

Comment:
This isn’t very accurate. Suggestion: "Return a dictionary that maps a chromosome name to the chromosome’s length. The mapping is read from `chr_lengths_file` if provided, and from the VCF header otherwise."

Reply:
Yeah this is better.